pkgs/by-name/wa/waybar/package.nix

```diff
@@ -3,46 +3,55 @@
   stdenv,
   bash,
   fetchFromGitHub,
-  SDL2,
-  alsa-lib,
-  catch2_3,
-  fftw,
-  glib,
+  fetchpatch,
+
+  # nativeBuildInputs
+  meson,
+  ninja,
+  pkg-config,
+  wayland-scanner,
+  wrapGAppsHook3,
   gobject-introspection,
+  scdoc,
+
+  # propagatedBuildInputs
+  glib,
+  playerctl,
+  python3,
+
+  # buildInputs
   gtk-layer-shell,
   gtkmm3,
   howard-hinnant-date,
-  hyprland,
-  iniparser,
   jsoncpp,
-  libdbusmenu-gtk3,
+  libsigcxx,
+  libxkbcommon,
+  spdlog,
+  wayland,
+  alsa-lib,
+  fftw,
+  iniparser,
+  ncurses,
+  portaudio,
+  SDL2,
   libevdev,
-  libinotify-kqueue,
+  hyprland,
   libinput,
   libjack2,
   libmpdclient,
   libnl,
   libpulseaudio,
-  libsigcxx,
-  libxkbcommon,
-  meson,
-  ncurses,
-  ninja,
-  pipewire,
-  pkg-config,
-  playerctl,
-  portaudio,
-  python3,
-  scdoc,
   sndio,
-  spdlog,
   sway,
+  libdbusmenu-gtk3,
   udev,
   upower,
-  wayland,
-  wayland-scanner,
   wireplumber,
-  wrapGAppsHook3,
+  pipewire,
+  libinotify-kqueue,
+
+  # nativeCheckInputs
+  catch2_3,
```

Comment:
No need to change the order, we can see what are in nativeBuildInputs below, or in buildInputs.
stdenv and lib can stay at the top, keep/rearrange the rest in alphabetical order.

Reply:
Sure, I reverted this change.